waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/mapping/service/PanopticOperationHandler.java

```diff
@@ -51,9 +51,9 @@
   private static final Logger LOG = LoggerFactory.getLogger(PanopticOperationHandler.class);
   private static final String INTERRUPTED_MESSAGE = "Execution was interrupted: ";
   private static final String SLOW_METASTORE_MESSAGE = "Metastore {} was slow to respond so results are omitted";
-  private static final long SET_UGI_TIMEOUT = 5000;
-  private static final long GET_DATABASES_TIMEOUT = 6000;
-  private static final long GET_TABLE_META_TIMEOUT = 400;
+  private static final long SET_UGI_TIMEOUT = 400;
```

Comment:
Let's make these: `TimeUnit.MILLISECONDS.toMillis(....)` So we got units documented.